index.bs

```diff
@@ -140,6 +149,25 @@ The <dfn attribute for="Document">prerendering</dfn> getter steps are to return
 The <dfn attribute for="Document">onprerenderingchange</dfn> attribute is an [=event handler IDL attribute=] corresponding to the <dfn event for="Document">prerenderingchange</dfn> [=event handler event type=].
 
 
+<h2 id="prerender-processing-model">Prerender processing model</h2>
```

Comment:
Let's delete this heading, and just move the sentence "Add a new..." into the "Link type prerender" section? Compare to https://jeremyroman.github.io/alternate-loading-modes/#prerendering-bcs